### ISSUE TITLE
kendo-draggable - Type error: 'bindTo', which lacks return-type annotation, implicitly has an 'any' return type. (Typescript Typings issue)

### ISSUE THREAD

**Issue by Beej126** [telerik/kendo-react#168]
## I'm submitting a...
* Bug report 

## Current behavior
.../ClientApp/node_modules/@telerik/kendo-draggable/dist/npm/main.d.ts
Type error: 'bindTo', which lacks return-type annotation, implicitly has an 'any' return type.  TS7010

line | code
-- | --
8 | export default class Draggable {
9 |     constructor(options?: DraggableOptions)
10 |     bindTo(element: Element)
error     |     ^
11 |     destroy()
12 | }
13 |

## Easy Fix
I just added ": void" return-type declarations to both bindTo() and destroy() in my local copies and it compiled and ran fine.
```
export default class Draggable {
    constructor(options?: DraggableOptions)
    bindTo(element: Element): void
    destroy(): void
}
```

## Minimal reproduction of the problem with instructions
In a Create-React-App 2.1+ Typescript project just import kendo-react-dialogs in a tsx page. Don't even have to use it.
```
import { Dialog, DialogActionsBar } from '@progress/kendo-react-dialogs';
```

## Environment

Package versions:
```
+-- @progress/kendo-data-query@1.5.0
+-- @progress/kendo-react-animation@2.6.1
+-- @progress/kendo-react-dateinputs@2.6.1
+-- @progress/kendo-react-dialogs@2.6.1
+-- @progress/kendo-react-dropdowns@2.6.1
+-- @progress/kendo-react-excel-export@2.6.1
+-- @progress/kendo-react-grid@2.6.1
+-- @progress/kendo-react-inputs@2.6.1
+-- @progress/kendo-react-intl@2.6.1
+-- @progress/kendo-react-layout@2.6.1
+-- @progress/kendo-react-popup@2.6.1
+-- @progress/kendo-theme-bootstrap@2.18.0
+-- @progress/kendo-theme-default@2.63.0
+-- @types/jest@23.3.10
+-- @types/lodash@4.14.118
+-- @types/node@10.12.12
+-- @types/react@16.7.11
+-- @types/react-dom@16.0.11
+-- @types/react-router@4.4.1
+-- @types/react-router-dom@4.3.1
+-- @types/toastr@2.1.35
+-- bootstrap@4.1.3
+-- change-case@3.0.2
+-- dexie@2.0.4
+-- jquery@3.3.1
+-- lodash@4.17.11
+-- moment@2.22.2
+-- node-sass@4.10.0
+-- npm-run-all@4.1.5
+-- popper.js@1.14.6
+-- react@16.6.3
+-- react-dom@16.6.3
+-- react-router@4.3.1
+-- react-router-dom@4.3.1
+-- react-scripts@2.1.1
+-- toastr@2.1.4
`-- typescript@3.2.1
```

Browser:
- Chrome (desktop) version: 71.0.3578.98 (Official Build) (64-bit)

System:
- TypeScript version: 3.1.3
- Node version: 10.13.0
- npm version: 6.4.1
- Platform:  Windows 10

**Comment by PaitoAnderson** [comment]
I have this same error as well.
```
ERROR in [at-loader] ./node_modules/@telerik/kendo-draggable/dist/npm/main.d.ts:10:5
    TS7010: 'bindTo', which lacks return-type annotation, implicitly has an 'any' return type.

ERROR in [at-loader] ./node_modules/@telerik/kendo-draggable/dist/npm/main.d.ts:11:5
    TS7010: 'destroy', which lacks return-type annotation, implicitly has an 'any' return type.
```

**Comment by tsvetomir** [comment]
Fix is underway, apologies for the delay. In the meanwhile, you can workaround the issue by turning off the `noImplicitAny` flag in the projects `tsconfig.json` file:

```json
"compilerOptions": {
  "noImplicitAny": false
}
```

**Comment by tsvetomir** [comment]
Fixed in @telerik/kendo-draggable v1.9.1
